File: Day_2/program3.py
def CheakDivisible(No):
	if No % 5==0:return True
	else:return False

def main():

	No = int(input("Enter the number..."))
	Ret = CheakDivisible(No)
	if Ret :print("Number is divisible by 5")
	else:print("Not divisible by 5")
	

if __name__ == "__main__":
	main()

File: Day_2/program3.cpp
#include<iostream>
using namespace std;
int CheakDivisible(int No)
{
	if( No % 5 == 0)
	{
		return 1;
	}
	else
	{
		return 2;	
	}
}


int main()
{
	int No =0;
	cout<<"Enter the number:\n";
	cin>>No;
	int iRet = CheakDivisible(No);
	if(iRet == 1)
	{
		cout<<"The number divisible by 5\n";
	}
	else
	{
		cout<<"The number is not divisible by 5\n";
	}
	return 0;
}
